src/modules/mc_nn_control/mc_nn_control_params.c

```diff
@@ -62,7 +62,7 @@ PARAM_DEFINE_INT32(MAX_RPM, 22000);
  * @max 80000
  * @group Neural Control
  */
-PARAM_DEFINE_INT32(MIN_RPM, 1000);
+PARAM_DEFINE_INT32(NN_MIN_RPM, 1000);
```

Comment:
Thanks for all these. Where they appear in the docs, can you please create links as shown here https://github.com/PX4/PX4-Autopilot/pull/24366/files/72e8a0e149b2c166ec15a37e40f270cfa9fb04e6..714ed46f114d8d96e70ee4e1f2c69aa59febc358#r2157735333